libs/mocks/build.gradle

```diff
@@ -12,11 +12,10 @@ android {
 
 dependencies {
     implementation("com.github.tomakehurst:wiremock:$wiremockMocksVersion") {
-        exclude group: 'com.fasterxml.jackson.core', module: 'jackson-core'
```

Comment:
@ParaskP7 Do you know why this `exclude` was necessary previously but now it's not?

Reply:
Sounds good to me.